rails/app/controllers/home_controller.rb

```diff
@@ -3,6 +3,11 @@ class HomeController < ApplicationController
   def index
   end
 
+  def community_search_index
```

Comment:
gosh. so this just gives us the skeleton, and doesn't quite get to switching between or setting up home controller right for switching between communities?


Upon reading of the code (i read the rb and skipped the html, which appeared to be mostly styling changes)
This makes me concerned that if we had a deployment already out there, it would mess that up. Am i misreading that or ...? (because users who are not superusers would not have community ids associated with them)

Reply:
It does switch communities. A user can’t belong to more than one community so it doesn’t need to switch. Everything works as it did previously for a single user in the system... so long as that user belongs to a community. Which given how we set everything up each time, that should be the case.

As for the case where a user does not have a community, then it does get a little gnarly. We hope to have search setup in the future that allows users not associated with a community to search and see publicly available stories on a map.

All super users do is push you into a special admin dashboard for creating communities. They actually won’t have communities. Regular users will, which get directed as normal, and then users without a community get shoved into the unfinished search.

I highly recommend checking out the deployment to see it in action